packages/server/api/src/app/ai/config/ai-config.controller.ts

```diff
@@ -15,7 +16,26 @@ export const aiConfigController: FastifyPluginAsyncTypebox = async (app) => {
     '/',
     SaveAiConfigOptions,
     async (request, reply): Promise<AiConfig> => {
-      const { valid, error } = await validateAiProviderConfig(request.body);
+      let existingApiKey = request.body.apiKey;
+      if (request.body.id) {
+        const exisiting = await aiConfigService.get(
+          {
+            projectId: request.principal.projectId,
+            id: request.body.id,
+          },
+          false,
```

Comment:
what is this false?

Reply:
default the get method returns **REDACTED** with this flag it returns me the actual apiKey

Reply:
I think it's very unclear, it would be better to have a separate method, i.e. getWithApiKey

Reply:
Best i cant offer is adding it to params, so you have to write { getRedacted: false }

Reply:
nvm i can not offer that, but if you look at the function signature its clear. If you want we can add a cleanup ticket for it either way i dont think its relevant fixing it right now.

Reply:
getActiveConfig gets the currently enabled config. get config gets the config based on ID.